app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -22,7 +22,8 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentDao study
         try {
             Study study = studyService.getStudy("api");
             try {
-                if (study.getUserProfileAttributes().isEmpty()) {
+                if (study.getUserProfileAttributes().size() < 2) {
```

Comment:
Could it be more robust if we check against the expected number of attributes. If not the same as expected, we clear and add all of them.